### PR TITLE
Always run dnsmasq

### PR DESCRIPTION
**Pull request by gabriel-v**

### CODE REVIEW

File: ansible/roles/discover/templates/local.py [hunk 1]
```diff
@@ -7,5 +7,4 @@
 DNSMASQ_INTERFACE = None
 
 # Flask debug
-DEBUG = False
-# Flask secret key
+DEBUG = {% if devel %}True{% else %}False{% endif %}
```
Comment (mgax) [inline ansible/roles/discover/templates/local.py]: Yep.

File: test/zeroconf/node-test.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 import subprocess
 
 def dig_for(hostname):
-    proc = subprocess.run(['dig', hostname, '@localhost', '+short'], check=False, stderr=subprocess.STDOUT, stdout=subprocess.PIPE)
+    proc = subprocess.run(['dig', hostname, '@localhost', '+short'], check=True, stderr=subprocess.STDOUT, stdout=subprocess.PIPE)
```
Comment (mgax) [inline test/zeroconf/node-test.py]: Isn't `check=True` the default?

Comment (gabriel-v) [reply test/zeroconf/node-test.py]: No, it's not. https://docs.python.org/3/library/subprocess.html#subprocess.run

Comment (mgax) [reply test/zeroconf/node-test.py]: :+1:

**Pull request merged**